### PR TITLE
feat: production forecasting using exponential smoothing method

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
## Production Forecasting (Yearly)
image

<details><summary>Half Yearly Production Forecasting</summary><br>

Screenshot 2020-05-14 at 8 55 12 AM

</details>

<details><summary>Quarterly Production Forecasting</summary><br>
Screenshot 2020-05-14 at 9 13 40 AM

</details>

<details><summary>Monthly Production Forecasting</summary><br>

Screenshot 2020-05-14 at 8 54 52 AM

</details>

### CODE REVIEW

File: erpnext/manufacturing/report/production_forecasting/production_forecasting.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+ "add_total_row": 0,
+ "creation": "2020-05-11 14:22:26.866107",
+ "disable_prepared_report": 0,
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "idx": 0,
+ "is_standard": "Yes",
+ "letter_head": "Gadgets International",
```
Comment (rmehta) [inline erpnext/manufacturing/report/production_forecasting/production_forecasting.json]: letterhead ?

Comment (rohitwaghchaure) [reply erpnext/manufacturing/report/production_forecasting/production_forecasting.json]: Fixed

**Comment by rohitwaghchaure** [comment]
## Added Forecast Based On
Screenshot 2020-05-15 at 5 14 47 AM

**Comment by rohitwaghchaure** [comment]
## Renamed to Exponential Smoothing Forecasting

Screenshot 2020-05-15 at 5 21 35 AM

**Comment by rohitwaghchaure** [comment]
## Added Based On Document and Based On Field (Qty/Amount)
Screenshot 2020-05-15 at 6 02 26 AM

**Comment by parsimonyERP** [comment]
+1

**Comment by rohitwaghchaure** [comment]
Screenshot 2020-05-20 at 12 08 14 PM

**Pull request merged**